projects/step-core/src/lib/modules/table/shared/parameters-filter-condition.ts

```diff
@@ -0,0 +1,36 @@
+import { FilterCondition } from './filter-condition';
+import { TableCollectionFilter, TableRequestFilter, Regex as RegexFilter } from '../../../client/step-client-module';
+import { CompareCondition } from '../../basics/step-basics.module';
+
+export class ParametersFilterCondition extends FilterCondition {
+  constructor(private availableParameters: ReadonlyArray<string>, private searchValue?: string) {
+    super();
+  }
+
+  override isEmpty(): boolean {
+    return !this.searchValue;
+  }
+
+  override toRequestFilter(field: string): Array<TableRequestFilter | undefined> {
+    if (!this.searchValue || this.availableParameters.length === 0) {
```

Comment:
I know this ```.length === 0``` is considered more explicit, but JavaScript specifically gives us the ability to work with truthy and falsy values in order to simplify the code, why not just write ```!this.availableParameters.length```

Reply:
Fixed